src/core/core.controller.js

```diff
@@ -147,8 +147,8 @@ function updateConfig(chart) {
 	chart.tooltip.initialize();
 }
 
-function positionIsHorizontal(position) {
-	return position === 'top' || position === 'bottom';
+function positionIsHorizontal(position, axis = undefined) {
+	return position === 'top' || position === 'bottom' || (position === 'center' && axis === 'x');
```

Comment:
Should `axis === 'x' ` alone be enough? (future ready and simpler)

Reply:
Yes, that should work. I've updated to that